t/api/04-jobs.t

```diff
@@ -698,6 +698,17 @@ subtest 'TEST is only mandatory parameter' => sub {
     $t->json_is('/job/settings/DISTRI'  => undef);
 };
 
+subtest 'Job with SCENARIO' => sub {
+    $jobs_post_params{SCENARIO} = 'foo';
+    $t->post_ok('/api/v1/jobs', form => \%jobs_post_params)->status_is(200, 'posted job with scenario');
+    is(
+        $jobs->find($t->tx->res->json->{id})->settings_hash->{NAME},
+        '00099995-opensuse-Tumbleweed-DVD-aarch64-Build1234-foo@64bit',
+        'scenario reflected in job name'
```

Comment:
here we can see a semantic problem. The scenario name is actually – following http://open.qa/docs/#concepts – "opensuse-Tumbleweed-DVD-aarch64-foo@64bit", not just "foo". In another case you used "testsuite_name" or "job_templates_name". How about we use "job_templates_name" consistently instead?